Tasks.md

```diff
@@ -1,94 +1,301 @@
+    console.log('Whackamole!')
+}
+
+// ...
+
+moles[0].addEventListener('click', whack)
+```
+</details>
+
+
+### 5) WHACK with CSS
+- optional: remove the log inside of `whack()`
+- when you whack a mole, run `hide()` on that mole
+    - hint: `whack()` receives an `event` object as parameter. This event has a `target` property, which is the element that was clicked, which is the mole!
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+```javascript
+function whack(event) {
+    const mole = event.target
+    hide(mole)
+}
+```
+</details>
+
+
+### 6) HTML
+- in the HTML, copy/paste the hole `<div>` to make 6 holes, one after the other
+-  add event listeners to ALL the moles in the javascript code
+    - hint: use a `for` loop
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+*HTML*
+```html
+<div class="game">
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+    <div class="hole">
+        <div class="mole"></div>
+    </div>
+</div>
+```
+
+*Javascript*
+```javascript
+// ...
+
+for (let mole of moles) {
+    mole.addEventListener('click', whack)
+}
+```
+</details>
+
+
+### 7) RANDOM MOLE
+- above `startGame()`, write a function that returns a random mole (i.e.: a random element from the moles array)
+    - think about a good name for this function
+    - first select a random index: `Math.floor(Math.random() * moles.length)`
+    - then access that index of the `moles` array
+    - return that element
+- inside `peep()`, call the function you created and assign it to the `mole` variable
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+```javascript
+// returns a random element from the holes array
+function randomMole() {
+    const index = Math.floor(Math.random() * moles.length)
+    const mole = moles[index]
+    return mole
+}
+
+async function peep() {
+    const mole = randomMole()
+    // ...
+}
+```
+</details>
+
+
+### 8) SCORE
+- in the beginning of the file, add a variable for the score
+    - question: do we use `const` or `let`?
+- set it to `0` when the game starts, before the call to `peep()`
+- show it when the game starts (update the HTML `scoreBoard` element, use the `textContent` property)
+- increase it by 1 when you wack a mole (i.e. in the `whack()` function)
+    - hint: first, write the expression for "your score plus 1 point". Then assign this expression to the score-keeping variable
+- show the new score immediately, like we did in `startGame()`
+
+<details>
+<summary>Feeling lost? Expand to get a hint!</summary>
+
+```javascript
+let score
+
+// ...
+
+function startGame() {
+    score = 0
+    scoreBoard.textContent = score
+    // ...
+}
```

Comment:
This is great, because we should always add code on top (or on the bottom) of what's already there, not in the middle. But in this case we need to change `08.js` (and also `09` and `10`) so that they have the same order of statements:
```javascript
// starts the game
function startGame() {
    score = 0
    scoreBoard.textContent = score
    startButton.disabled = true
    peep()
    setInterval(peep, 1400)
}
```
could you change that?